website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -3,321 +3,102 @@ title: Helm Based Operator Quickstart
-in `watches.yaml` and executes Helm releases using the specified chart:
-
-```yaml
-# Use the 'create api' subcommand to add watches to this file.
-- group: example.com
-  version: v1alpha1
-  kind: Nginx
-  chart: helm-charts/nginx
-# +kubebuilder:scaffold:watch
-```
-
-### Reviewing the Nginx Helm Chart
-
-When a Helm operator project is created, the SDK creates an example Helm chart
-that contains a set of templates for a simple Nginx release.
-
-For this example, we have templates for deployment, service, and ingress
-resources, along with a `NOTES.txt` template, which Helm chart developers use
-to convey helpful information about a release.
-
-If you aren't already familiar with Helm Charts, take a moment to review
-the [Helm Chart developer documentation][helm-charts].
-
-### Understanding the Nginx CR spec
-
-Helm uses a concept called [values][helm-values] to provide customizations
-to a Helm chart's defaults, which are defined in the Helm chart's `values.yaml`
-file.
-
-Overriding these defaults is as simple as setting the desired values in the CR
-spec. Let's use the number of replicas as an example.
-
-First, inspecting `helm-charts/nginx/values.yaml`, we see that the chart has a
-value called `replicaCount` and it is set to `1` by default. If we want to have
-2 nginx instances in our deployment, we would need to make sure our CR spec
-contained `replicaCount: 2`.
-
-Update `config/samples/example_v1alpha1_nginx.yaml` to look like the following:
-
-```yaml
-apiVersion: example.com/v1alpha1
-kind: Nginx
-metadata:
-  name: nginx-sample
-spec:
-  replicaCount: 2
-```
-
-Similarly, we see that the default service port is set to `80`, but we would
-like to use `8080`, so we'll again update `config/samples/example_v1alpha1_nginx.yaml`
-by adding the service port override:
-
-```yaml
-apiVersion: example.com/v1alpha1
-kind: Nginx
-metadata:
-  name: example-nginx
-spec:
-  replicaCount: 2
-  service:
-    port: 8080
-```
+## Creating an API
 
-As you may have noticed, the Helm operator simply applies the entire spec as if
-it was the contents of a values file, just like `helm install -f ./overrides.yaml`
-works.
-
-## Build and run the operator
-
-Before running the operator, Kubernetes needs to know about the new custom
-resource definition the operator will be watching.
-
-Deploy the CRD:
+First, we will use [`helm repo add`][helm-repo-add] to configure its repository.
 
 ```sh
-make install
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 ```
 
-Once this is done, there are two ways to run the operator:
-
-- As a pod inside a Kubernetes cluster
-- As a go program outside the cluster using `operator-sdk`
-
-### 1. Run as a pod inside a Kubernetes cluster
-
-Running as a pod inside a Kubernetes cluster is preferred for production use.
-
-Build the nginx-operator image and push it to a registry:
+Let's create a new API with the [Helm Chart Nginx Ingress][helm-chart-nginx]:
```

Comment:
These are extra unnecessary steps. Let's just use `operator-sdk create api --group demo --version v1 --kind Nginx`, which will generate a simple nginx chart. This simplifies the quickstart and is one fewer tool some needs to use (`helm`) since there's no need to configure a chart repo.